conf/uppmax-devel.config

```diff
@@ -14,6 +14,17 @@ executor {
 }
 process {
   queue = 'devel'
+  container = 'file:///home/matq007/private/pipeline_images/rnafusion-dev.simg'
```

Comment:
You can add a params like containerPath.
cf: https://github.com/SciLifeLab/Sarek/blob/e504b0e7c1f025641469b76edbc64e4a04bdb009/conf/singularity-path.config

Reply:
Ah, this is much better solution, thank you. I'll include this one 👍